deploy_preservica_config.tf

```diff
@@ -31,6 +32,23 @@ module "preservica_config_sns" {
   }
 }
 
+module "dr2_preservica_config_sns" {
+  source = "git::https://github.com/nationalarchives/da-terraform-modules//sns"
+  sns_policy = templatefile("${path.module}/templates/sns/s3_notifications_policy.json.tpl", {
+    topic_name  = "${local.environment}-preservica-config"
+    bucket_name = "${local.environment}-dr2-preservica-config"
```

Comment:
Should this be `local.preservica_config_bucket_name`?

Reply:
Good spot, yep.